lib/chef-acceptance/chef_client_run.rb

```diff
@@ -3,42 +3,35 @@
 require 'json'
 require 'bundler'
 require 'chef-acceptance/acceptance_cookbook'
-require 'chef-acceptance/shellout_builder'
 
 module ChefAcceptance
-  class ChefRunner
+
+  # Responsible for generating a CCR shellout and running it
+  class ChefClientRun
```

Comment:
I think we should rename this class (again).  `ChefClientCommand` or `ChefClientShellout`?  Something that indicates that it initializes the chef client and `can` execute the chef client.